pkgs/tools/security/yara/default.nix

```diff
@@ -1,29 +1,34 @@
-{ lib, stdenv
-, fetchFromGitHub
+{ lib
+, stdenv
 , autoreconfHook
+, fetchFromGitHub
+, file
+, jansson
+, openssl
 , pcre
 , pkg-config
 , protobufc
-, withCrypto ? true, openssl
-, enableCuckoo ? true, jansson
-, enableDex ? true
-, enableDotNet ? true
-, enableMacho ? true
-, enableMagic ? true, file
-, enableStatic ? false
 }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "yara";
-  version = "4.3.1";
+  version = "4.3.2";
 
   src = fetchFromGitHub {
     owner = "VirusTotal";
-    repo = pname;
-    rev = "v${version}";
-    hash = "sha256-Q+Q52W/MhurG3x0CIr0nv31qc4bdaLDk9AGGpMxKOcI=";
+    repo = finalAttrs.pname;
+    rev = "refs/tags/v${finalAttrs.version}";
+    hash = "sha256-TmMHn/Zi/ZyJGHobEg9r1uLWu7e+8XJPwCvnUhQlD6I=";
   };
 
+  withCrypto = true ;
```

Comment:
apologies for the question

is this change because its preferred to use `overrideAttrs` over `override` ?

Reply:
I'm also wondering the same question. I have the feeling that this is not what it is supposed to be done.

Reply:
@fabaff ^^